Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:xenial
+LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
+
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends bzip2 ca-certificates curl file fonts-dejavu-core g++ git locales make openssh-client patch sudo uuid-runtime \
```

Comment:
Can this be split into a few lines? Maybe even one per line?

Reply:
it might increase the size https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better

Reply:
it can still be a single `RUN` statement, but the packages can be written on separate lines with `\`; I think that's what @MikeMcQuaid was suggesting

Reply:
Done.